### PR TITLE
Manage multiple line expressions

### PR DESCRIPTION
**Pull request by guillett**

### CODE REVIEW

**Comment by guillett** [comment]
A solution to solve https://circleci.com/gh/openfisca/openfisca-france/2907?utm_campaign=build-failed&utm_medium=email&utm_source=notification

File: biryani/baseconv.py [hunk 1]
```diff
@@ -578,6 +578,8 @@ def make_anything_to_float(accept_expression = False):
     (42.0, None)
     >>> make_anything_to_float(accept_expression = True)(u'(42 / 42 + 1) * 42 - 42')
     (42.0, None)
+    >>> make_anything_to_float(accept_expression = True)(u'(42 / 42 + 1)\n* 42 - 42')
```
Comment (Anna-Livia) [inline biryani/baseconv.py]: Great PR 🙌 
Sadly the doctstrings tests don't pass, due to the fact that they are strings, and therefore, they see `/n`as a newline in the docstring.
In order for this docstring to pass, you can write : 
`make_anything_to_float(accept_expression = True)(u'(42 / 42 + 1)\\n* 42 - 42')`

File: biryani/baseconv.py [hunk 2]
```diff
@@ -629,6 +631,8 @@ def make_anything_to_int(accept_expression = False):
     (42, None)
     >>> make_anything_to_int(accept_expression = True)(u'(42 / 42 + 1) * 42 - 42')
     (42, None)
+    >>> make_anything_to_int(accept_expression = True)(u'(42 / 42 + 1)\n* 42 - 42')
```
Comment (Anna-Livia) [inline biryani/baseconv.py]: Same here

File: biryani/baseconv.py (hunk 1 above)
Comment (guillett) [reply biryani/baseconv.py]: Thanks @Anna-Livia. Can you explicit the command you run that is failing?

_This repository misses documentation._

Comment (guillett) [reply biryani/baseconv.py]: In a virtual env with after `pip install --editable .[dev]`, `make test` is failing a lot.

Comment (guillett) [reply biryani/baseconv.py]: Merci thanks @Anna-Livia for those details!

**Comment by guillett** [comment]
Fix applied.

**Pull request merged**